.circleci/config/jobs/@packages.yml

```diff
@@ -116,6 +116,7 @@ macos-standalone:
       type: string
   macos:
     xcode: "13.2.1" # macOS 11.6.2 (Big Sur)
```

Comment:
Is the comment still relevant?

Reply:
should be, I haven't actually updated xcode/macos version